LayoutTests/fast/scrolling/mac/scrollend-event-scroll-snap.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html> <!-- webkit-test-runner [ AsyncOverflowScrollingEnabled=true ] -->
+<html>
+<head>
+    <style>
+        body {
+            height: 2000px;
+        }
+        .scroller {
+            scroll-snap-type: both mandatory;
+            position: absolute;
+            top: 310px;
+            left: 10px;
+            height: 300px;
+            width: 300px;
+            border: 20px solid gray;
+            overflow: scroll;
+        }
+        .content {
+            width: 200%;
+            height: 300%;
+        }
+        .target {
+            width: 200px;
+            height: 800px;
+            scroll-snap-align: start;
+        }
+        #space {
+            width: 2000px;
+            height: 2000px;
+        }
+        
+    </style>
+    <script src="../../../resources/js-test-pre.js"></script>
```

Comment:
You should use `js-test.js` instead of the pre/post variants

Reply:
Fixed.